server/src/test/java/org/apache/druid/initialization/ServerConfigTest.java

```diff
@@ -51,13 +54,27 @@ public void testSerde() throws Exception
         defaultConfig.getUnannouncePropagationDelay(),
         defaultConfig.getInflateBufferSize(),
         defaultConfig.getCompressionLevel(),
-        true
+        true,
+        ImmutableList.of("OPTIONS")
```

Comment:
Very small nit: `HttpMethod.OPTIONS` is better than the bare string.

Reply:
Done